server/src/test/java/org/apache/kafka/server/metrics/ClientMetricsTestUtils.java

```diff
@@ -59,6 +60,7 @@ public static RequestContext requestContext() throws UnknownHostException {
             new RequestHeader(ApiKeys.GET_TELEMETRY_SUBSCRIPTIONS, (short) 0, "producer-1", 0),
             "1",
             InetAddress.getLocalHost(),
+            Optional.of(56078),
```

Comment:
Could we define a constant for 56078 and reuse?

Reply:
Done.